components/sync_manager/metrics.yaml

```diff
@@ -28,6 +28,7 @@ sync_v2:
       - technical
     notification_emails:
       - sync-team@mozilla.com
+      - skhamis@mozilla.com
```

Comment:
nit: Aren't you already included in [sync-team@mozilla.com(https://groups.google.com/a/mozilla.com/g/sync-team/members)?

Reply:
Yup! If it's permanently monitored we want a single owner who can take responsibility 😬